internal/httphandler/ledger_request_handler_test.go

```diff
@@ -281,15 +284,16 @@ func TestBlockQuery(t *testing.T) {
 			}
 
 			if tt.expectedResponse != nil {
-				var res proto.Message
+				var res protoreflect.ProtoMessage
```

Comment:
This is probably because you had the wrong import, I assume if you import the new version we could continue to use proto.Message?